apps/api/src/app/bridge/bridge.controller.ts

```diff
@@ -37,7 +37,7 @@ import { CreateBridgeResponseDto } from './dtos/create-bridge-response.dto';
 export class BridgeController {
   constructor(
     private syncUsecase: Sync,
-    private validateBridgeUrlUsecase: GetBridgeStatus,
+    private getBridgeStatus: GetBridgeStatus,
```

Comment:
All HTTP requests to Bridge endpoints now occur in `ExecuteBridgeRequest` to centralize error handling.